doc/kubernetes/modules/ROOT/pages/installation-openshift.adoc

```diff
@@ -66,6 +66,14 @@ http://grafana.apps.**<domain name>**
 NOTE: This is a shared resource for all users using the OpenShift cluster.
 While the next section describes how to install multiple Keycloaks in different namespaces, this doesn't apply to the Grafana instance.
 
+== Openshift Logging
+Openshift logging is enabled by default. All application and infrastructure pod logs are stored in a non-replicated
```

Comment:
The way you set this up is that OpenShift logging is only enabled after "task monitoring" has been executed, therefore it is not enabled by default. 

Do we want it to be enabled by default? Then it should probably be called from "rosa_create_cluster.sh".

Reply:
Argh my bad, I made the assumption that the monitoring task was always called. I've updated it so that logging is installed as part of `rosa_create_cluster.sh` as I assume we will want this for all benchmark runs in the event of unexpected states being encountered.